tests/generic_config_updater/patch_sorter_test.py

```diff
@@ -3034,6 +3034,9 @@ def test_patch_sorter_success(self):
         data = Files.PATCH_SORTER_TEST_SUCCESS
         skip_exact_change_list_match = False
         for test_case_name in data:
+            # DPB_1_TO_4__SUCCESS will fail when unique_lanes is set to False
```

Comment:
>fail [](http://example.com/codeflow?start=39&length=4)

Why one less validator will make original passing testcase fail?

Reply:
You can check above sorting output here https://github.com/sonic-net/sonic-utilities/pull/2343#issuecomment-1232568895 
When lane validator is removed, the sorting order is changed. That's why it fails.